### PR TITLE
added docs on how to add github handle

### PR DESCRIPTION
**Pull request by Bullrich**
This was copied from https://github.com/polkadot-fellows/runtimes/pull/42

It adds an explanation to the readme on how to add your github handle to your identity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,3 +45,11 @@ They are:
 	- Example: `[{"address":"1e2345...",rank: 4, "githubHandle":"user-1"},{"address":"1e4532...",rank: 3, "githubHandle":"user-2"},{"address":"1e8335...",rank: 6}]`
 - `github-handles`: All the fellows handles separated by commas
 	- Example: `user-1,user-2,user-3`
+
+## Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics). See the following image for an example:
```
Comment (rzadp) [inline README.md]: Idea to consider:

You can direct the user to a link like this:
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics/decode/0x1c010407676974687562123c67697468756220757365726e616d653e0000000000000000

Then instruct the user to switch to the `Submission` - then you have the transaction ready, only the github username have to be changed.

File: README.md [hunk 2]
```diff
@@ -45,3 +45,11 @@ They are:
...
+
+![Add github name to the additional fields of the on-chain identity](.github/github-on-chain-identity-process.jpg)
+
+You should also at least add the `display` field. If you want to add `utf8` characters, you need to hex encode them and put the hex string (`0x` prefixed) into the field.
```
Comment (rzadp) [inline README.md]: > You should also at least add the `display` field.

Why? I mean I tried it out without that and it allowed me to do it, why is it needed in this action?

Screenshot 2023-09-29 at 14 52 29

Comment (rzadp) [inline README.md]: It's probably worth mentioning that if the user has some of those fields already configured on-chain, then sending a transaction with `None` will override those fields to an empty value.

So you cannot just add the github username, you have to include all other already-existing field values as well.

**Review by rzadp — APPROVED** [review]

**Comment by andresilva** [comment]
Maybe we should link to it (once it's merged) instead of copying it, otherwise it can get outdated.

**Pull request merged**